templates/employee.json

```diff
@@ -1,365 +1,77 @@
+      "operation": "{{@ datatype @}}-list",
+      "supports_since": true,
+      "system": "{{@ system @}}",
+      "trace": {},
+      "type": "rest"
     },
     "transform": [
       {
-        "type": "dtl",
-        "rules": {
-          "default": [
-            [
-              "add",
-              "::properties",
-              [
-                "dict",
-                "url",
-                "employees"
-              ]
-            ],
-            [
-              "copy",
-              "sesam-id"
-            ],
-            [
-              "add",
-              "_old-id",
-              [
-                "first",
-                [
-                  "hops",
-                  {
-                    "datasets": [
-                      "{{@ system @}}-employee-share-insert f"
-                    ],
-                    "where": [
-                      [
-                        "eq",
-                        "_S.sesam-id",
-                        "f.sesam-id"
-                      ]
-                    ],
-                    "return": "f.id",
-                    "track-dependencies": false
-                  }
-                ]
-              ]
-            ],
-            [
-              "add",
-              "::payload",
-              [
-                "apply",
-                "payload",
-                "_S."
-              ]
-            ],
-            [
-              "discard",
-              [
-                "is-empty",
-                "_T.payload.id"
-              ]
-            ]
-          ],
-          "payload": [
-            [
-              "copy",
-              "*"
-            ],
-            [
-              "remove",
-              "_*"
-            ],
-            [
-              "remove",
-              "sesam-id"
-            ],
-            [
-              "if",
-              [
-                "is-not-empty",
-                "_P._T._old_id"
-              ],
-              [
-                "add",
-                "id",
-                "_T._old-id"
-              ]
-            ]
-          ]
-        }
-      },
-      {
-        "type": "rest",
-        "system": "{{@ system @}}",
-        "operation": "insert",
-        "replace_entity": false
+        "properties": {
+          "primary_key": "id",
+          "rest_system": "{{@ system @}}",
+          "share_dataset": "{{@ system @}}-{{@ datatype @}}-share"
+        },
+        "template": "transform-collect-rest",
+        "type": "template"
       },
       {
-        "type": "dtl",
         "rules": {
           "default": [
             [
               "copy",
-              "sesam-id"
-            ],
-            [
-              "merge-union",
-              "_S.response"
+              "*"
             ],
             [
               "add",
-              "_id",
-              "_T.id"
+              "$last-modified",
+              [
+                "datetime",
+                "_S.updated_at"
```

Comment:
If an employee has only been created (but not updated ever) is updated_at the same as created_at? Or is it a default value? Because if they are not equal in that scenario then the $last-modified logic will need to be checking created_at too.